src/main/java/com/flansmod/common/driveables/DriveableType.java

```diff
@@ -411,10 +412,37 @@ else if(split[0].equals("RotatedDriverOffset"))
 			{
 				seats[0].rotatedOffset = new Vector3f(Integer.parseInt(split[1]) / 16F, Integer.parseInt(split[2]) / 16F, Integer.parseInt(split[3]) / 16F);
 			}
+			else if(split[0].equals("DriverAimSpeed"))
+			{
+				seats[0].aimingSpeed = new Vector3f(Float.parseFloat(split[1]), Float.parseFloat(split[2]), Float.parseFloat(split[3]));
+			}
 			else if(split[0].equals("RotatedPassengerOffset"))
 			{
 				seats[Integer.parseInt(split[1])].rotatedOffset = new Vector3f(Integer.parseInt(split[2]) / 16F, Integer.parseInt(split[3]) / 16F, Integer.parseInt(split[4]) / 16F);
 			}
+			else if(split[0].equals("PassengerAimSpeed"))
+			{
+				seats[Integer.parseInt(split[1])].rotatedOffset = new Vector3f(Float.parseFloat(split[2]), Float.parseFloat(split[3]), Float.parseFloat(split[4]));
+			}
+			else if(split[0].equals("DriverLegacyAiming")){
+				seats[0].legacyAiming = Boolean.parseBoolean(split[1]);
+			}
+			else if(split[0].equals("PassengerLegacyAiming")){
+				seats[Integer.parseInt(split[1])].legacyAiming = Boolean.parseBoolean(split[2]);
+			}
+			else if(split[0].equals("DriverYawBeforePitch")){
+				seats[0].yawBeforePitch = Boolean.parseBoolean(split[1]);
+			}
+			else if(split[0].equals("PassengerYawBeforePitch")){
+				seats[Integer.parseInt(split[1])].yawBeforePitch = Boolean.parseBoolean(split[2]);
+			}
+			else if(split[0].equals("DriverTraverseSounds")){
+				seats[0].traverseSounds = Boolean.parseBoolean(split[1]);
+			}
+			else if(split[0].equals("PassengerTraverseSounds")){
+				seats[Integer.parseInt(split[1])].traverseSounds = Boolean.parseBoolean(split[2]);
```

Comment:
So we're counting 0, 1, 2... etc. rather than 1, 2, 3... Is that in line with how other passenger seat counters work?